rider-fsharp/src/main/resources/META-INF/plugin.xml

```diff
@@ -47,24 +47,28 @@
     <projectConfigurable parentId="Errors" instance="com.jetbrains.rider.settings.FSharpInspectionsPage" />
 
     <intentionAction>
+      <language>F#</language>
```

Comment:
Have you checked if the checks in `SendToFsiActionBase` are still required after this change?